_health-care/_js/_form.js

```diff
@@ -41,6 +41,41 @@ function extractMonetaryValueTransform(formData, arg) {
   return Number(extractTransform(formData, arg)).toFixed(2).toString();
 }
 
+function extractAddressNameTransform(formData, arg) {
+  return extractTransform(formData, arg + "[name]");
+}
+
+function extractAddressStreetTransform(formData, arg) {
+  return extractTransform(formData, arg + "[street]");
+}
+
+function extractAddressCityTransform(formData, arg) {
+  return extractTransform(formData, arg + "[city]");
+}
+
+function extractAddressStateTransform(formData, arg) {
+  return extractTransform(formData, arg + "[state]");
+}
+
+function extractAddressZipcodeTransform(formData, arg) {
+  return extractTransform(formData, arg + "[zipcode]");
+}
+
+function extractAddressPhoneTransform(formData, arg) {
+  return extractTransform(formData, arg + "[phone]");
+}
+
+function extractAddressWithFormatTransform(formData, arg) {
+  let name = extractAddressNameTransform(formData, arg);
+  let street = extractAddressStreetTransform(formData, arg);
+  let city = extractAddressCityTransform(formData, arg);
+  let state = extractAddressStateTransform(formData, arg);
+  let zip = extractAddressZipcodeTransform(formData, arg);
+  let phone = extractPhoneTransform(formData, arg);
```

Comment:
Or maybe this should be `extractAddressPhoneTransform`? I think you're getting one JS test failure due to this discrepancy, but then when I run it locally I get a different error after that...